doc_src/license.rst

```diff
@@ -188,3 +188,18 @@ Permission is hereby granted, free of charge, to any person obtaining a copy of
 The above copyright notice and this permission notice shall be included in all copies or substantial portions of the Software.
 
 THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY, FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
+
+License for musl libc
+-----------------
+
+``fish`` contains code derived from  `musl-libc <https://www.musl-libc.org>`_ to implement printf. This code is distributed under the terms of the MIT license.
```

Comment:
There's already a copy of the MIT license directly above this, the notes about musl-libc can just be added to the paragraph describing the elements using MIT-licensed sources.

Reply:
fixed